packages/utilities/gel-foundations/src/scripts/arabic.js

```diff
@@ -71,142 +71,142 @@ const arabicTypography = {
   },
   canon: {
     groupA: {
-      fontSize: '32',
-      lineHeight: '42',
+      fontSize: '24',
+      lineHeight: '32',
```

Comment:
Where do I find the 'Font Stacks & Styles spec' to check these values against?

Reply:
Checked this against the 'Reith Qalam DRAFT' tab in the spec document looks good. It says 'Values below for testing purposes' are these definitely the final values?

Reply:
I am assuming so, but we can always check with UX to be sure.

Reply:
UX have confirmed that these values are only marked as Draft because they haven't been tested on a browser yet. The values may have to be tweaked based on UX review, but it's OK for us to use the values as is for now.